README.md

```diff
@@ -1,13 +1,163 @@
+codelabs, served millions of web visitors, and supported over 100 events, from local meetups
+all the way up to Google I/O.
+
+This project has been implemented as a volunteer project by a small group of dedicated Googlers
+who care deeply about this kind of “learning by doing” approach to education.
+
+## What's special about this tool?
+
+* Powerful and flexible authoring flow via Google Docs
+* Optional support for authoring in Markdown text
+* Ability to produce interactive web or markdown tutorials without writing any code
+* Easy interactive previewing
+* Usage monitoring via Google Analytics
+* Support for multiple target environments (kiosk, web, markdown, offline, etc.)
+* Support for anonymous use - ideal for public computers at developer events
+* Looks great, with a responsive web implementation
+* Remembers where the student left off when returning to a codelab
+* Mobile friendly user experience
+
+## Can I use this to create my own codelabs and serve my own codelabs online?
+
+Yes, the claat tool and the serving mechanism can be used by anyone to author their
+own codelabs and to serve up their own codelabs on the web.
+
+If you're interested in authoring codelabs, join [codelab-authors group](https://groups.google.com/forum/#!forum/codelab-authors),
+which connects you with other authors and provides access to the
+[Codelab Formatting Guide](https://g.co/codelabs/guide).
+
+## Ok, how do I use it?
+
+1. Create a doc following the syntax conventions described in the [Codelab Formatting Guide](https://g.co/codelabs/guide). Here’s an [example doc](https://docs.google.com/document/d/1rpHleSSeY-MJZ8JvncvYA8CFqlnlcrW8-a4uEaqizPY/edit). Feel free to copy that doc as a starter template. Once you have your own source doc, note its DocId, which is the long string near the end of the URL (right after docs.google.com/document/d/).
+
+1. Make one or more changes and preview your codelab, using the preview app provided by Google. To preview a codelab, install the Preview Codelab Chrome extension in your browser. Now you can preview a codelab directly from the Google Doc view by clicking the Chrome extension’s button, which will open a new tab to display the preview. Alternatively, navigate manually to https://codelabs-preview.appspot.com/?file_id=<google-doc-id>
+
+1. Install the claat command -- see the [README in the claat directory](https://github.com/googlecodelabs/tools/blob/master/claat/README.md) of this repo for instructions..
+
+1. Run the claat command to transform the doc contents into one of the supported output formats. The default supported formats are html and markdown but the claat tool supports adding additional formats by specifying a path to a Go template. For example, using the example document above:
+
+    `$ claat export -f html 1rpHleSSeY-MJZ8JvncvYA8CFqlnlcrW8-a4uEaqizPY`
+    <br>`ok      your-first-pwapp`
+
+1. Run the claat serve command.
+
+    `$ claat serve`
```

Comment:
remove back quotes. 4 spaces indentation already does it.

Reply:
Same disp as previous.